var/spack/repos/builtin/packages/jedi-cmake/package.py

```diff
@@ -22,4 +22,7 @@ class JediCmake(CMakePackage):
     )
     version("1.3.0", commit="729a9b2ec97a7e93cbc58213493f28ca11f08754")
 
+    depends_on("c", type="build")  # generated
```

Comment:
jedi-cmake contains no C or C++ code. Just a bunch of cmake macros

Reply:
https://github.com/JCSDA/jedi-cmake/blob/master/CMakeLists.txt#L8 

https://cmake.org/cmake/help/latest/command/project.html

"By default C and CXX are enabled if no language options are given. Specify language NONE, or use the LANGUAGES keyword and list no languages, to skip enabling any languages."

Reply:
Oh, so that should be fixed upstream and new versions wouldn't depend on C/C++ anymore. Thanks for the explanation.